src/actions/delete_objects.rs

```diff
@@ -69,6 +70,9 @@ impl<'a, I> DeleteObjects<'a, I>
 where
     I: Iterator<Item = &'a ObjectIdentifier>,
 {
+    /// Generate the XML body for the request.
+    /// # Panics
```

Comment:
Looks better this way and markdown also prefers it.

```suggestion
    ///
    /// # Panics
    ///
```

Reply:
Fixed